components/html-block/html-block.js

```diff
@@ -9,7 +9,10 @@ export const htmlBlockContentStyles = css`
 	.d2l-html-block-compact {
 		font-size: 0.8rem;
 		font-weight: 400;
-		line-height: 1.2rem;
+		line-height: 1.5;
```

Comment:
1.5 = 1.2rem / 0.8rem

Reply:
In this particular case, there would be no difference under any circumstance, given that the font-size is based on `rem`, correct?

Reply:
In the case that the default font size is used, yes. Because `line-height` is based on the element's font, this does end up needing to be unitless to cover the case where someone changes the font-size on an element-level.